sh: 1: timeout.sh: not found
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 1166, in run
    self.function(*self.args, **self.kwargs)
  File "/home/aos/src/turretclock/mqttclient.py", line 28, in mqttTelemetry
    avgs = sqlaverages(drift)
  File "/home/aos/src/turretclock/mqttclient.py", line 113, in sqlaverages
    avg1d = (23*avg1d + (oldest/3600)*avg1d2)/(23+(oldest/3600))
TypeError: unsupported operand type(s) for *: 'int' and 'NoneType'

Traceback (most recent call last):
  File "./turretclock.py", line 148, in <module>
    c.loop()
  File "/home/aos/src/turretclock/commander.py", line 161, in loop
    self.eloop.run()
  File "/usr/lib/python3/dist-packages/urwid/main_loop.py", line 286, in run
    self._run()
  File "/usr/lib/python3/dist-packages/urwid/main_loop.py", line 384, in _run
    self.event_loop.run()
  File "/usr/lib/python3/dist-packages/urwid/main_loop.py", line 788, in run
    self._loop()
  File "/usr/lib/python3/dist-packages/urwid/main_loop.py", line 812, in _loop
    ready, w, err = select.select(fds, [], fds)
KeyboardInterrupt
Exception in thread ui:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/home/aos/src/turretclock/ui.py", line 279, in outputD
    c.output(item[0],cfg.ui_colors[item[1]])
  File "/home/aos/src/turretclock/commander.py", line 186, in output
    self.eloop.draw_screen()
  File "/usr/lib/python3/dist-packages/urwid/main_loop.py", line 587, in draw_screen
    self.screen.draw_screen(self.screen_size, canvas)
  File "/usr/lib/python3/dist-packages/urwid/raw_display.py", line 682, in draw_screen
    assert self._started
AssertionError

